scripts/calibrateArmDac.sh

```diff
@@ -34,71 +34,80 @@ then
     return
 fi
 
-echo -e "ChamberName\tscandate\tCFG_THR_ARM_DAC" 2>&1 | tee listOfScanDates_calibrateArmDac_${DETECTOR}.txt
+echo -e "ChamberName\tscandate\tCFG_THR_ARM_DAC" 2>&1 | tee ${DATA_PATH}/${DETECTOR}/armDacCal/${scandate}/listOfScanDates_calibrateArmDac_${DETECTOR}.txt
+
+scandate=$(date +%Y.%m.%d.%H.%M)
+mkdir -p ${DATA_PATH}/${DETECTOR}/armDacCal/${scandate}
+ln -s ${DATA_PATH}/${DETECTOR}/armDacCal/${scandate} ${DATA_PATH}/${DETECTOR}/armDacCal/current
 
 runNum=0
 for armDacVal in $(echo $LIST_ARM_DAC | sed "s/,/ /g")
 do
-    if [ -e scurveLog_ArmDac_${armDacVal}.txt ]
+    if [ -e ${DATA_PATH}/${DETECTOR}/armDacCal/${scandate}/scurveLog_ArmDac_${armDacVal}.txt ]
     then
-        rm scurveLog_ArmDac_${armDacVal}.txt
+        rm ${DATA_PATH}/${DETECTOR}/armDacCal/${scandate}/scurveLog_ArmDac_${armDacVal}.txt
```

Comment:
I think this block could be removed now.  It's no longer necessary since each call of this script should generate a new directory, and the first `tee` below (e.g. without option `-a`) will ensure it's correctly a fresh file.